package.json

```diff
@@ -49,7 +49,7 @@
     "test:typings:extension": "tsc --noEmit",
     "test:typings:webviews": "cd ./src/webviews && npm run type-check",
     "test:unit": "echo \"Unit testing not yet implemented.\"",
-    "test:e2e": "echo \"End-to-end testing not yet implemented.\""
+    "test:e2e": "wdio run ./test/wdio.conf.ts"
```

Comment:
Should we adjust L47 above to *not* contain `&& npm run test:e2e`? Or have it to only run if env var `CI` is set?

Ugh.. not being able to run those tests locally is such a bummer. What if we (for some scenarios?) would install radicle in specific destination e.g. under [`os.tempDir()`](https://nodejs.org/api/os.html#os_os_tmpdir) with

```sh
curl -sSf https://radicle.xyz/install | sh -s -- --prefix=my/temp/dir --no-modify-path
```

According to the installation script:

> `--prefix=PATH       Radicle install prefix (default: ~/.radicle)`

Reply:
OK, great.

Resolving for now, but I think following up to make it possible to run e2e tests locally (and thus also out of GH actions!!) is worth coming back for a follow-up, given that it seems technically doable.